actors/evm/src/interpreter/system.rs

```diff
@@ -346,6 +351,17 @@ impl<'r, RT: Runtime> System<'r, RT> {
             _ => Ok(EthAddress::from_id(actor_id)),
         }
     }
+
+    pub fn get_or_init_randomness(&mut self) -> Result<&[u8; 32], StatusCode> {
```

Comment:
Just call this `get_randomness`. The user doesn't care that it caches internally.